### ISSUE TITLE
Compatible with alien races.

### ISSUE THREAD

**Issue by BLkun** [Nibato/AutoExtractGenes#3]
Hello. Is it possible to add compatibility with races from other mods? Currently, the checkbox for automatic gene extraction is only available on vanilla races.

**Comment by Nibato** [comment]
It should work already with most mods that use [Humanoid Alien Races](https://steamcommunity.com/sharedfiles/filedetails/?id=839005762). Though sometimes you may need to load Auto Extract Genes after (or before) some or all of the race mods to get it working properly.

As far as races that don't use HAR, I unfortunately don't really have the time to dedicate to that at the moment. However, I wouldn't object to someone else contributing and putting in a pull request for it.

**Comment by BLkun** [comment]
No, putting it at the end of the list did nothing to improve the situation.
After asking in other threads, I was told that a compatibility patch for Harmony is needed (probably something in the mod code that Harmony cannot handle).
Thank you for trying to help, in the end, even a half-working mod is still better than not having it at all.
